radio/src/gui/480x272/lcd.cpp

```diff
@@ -280,7 +280,7 @@ void drawTimerMode(coord_t x, coord_t y, int32_t mode, LcdFlags att)
 {
   if (mode >= 0) {
     if (mode < TMRMODE_COUNT)
-      return lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
+      lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
```

Comment:
all examples you proposed are OK, I also don't like the one-line version.
also I prefer the brackets when there is a need for an comment like in this example:
if (test) {
 // I explain why I return here
 return ...
}